src/main/java/com/onepagecrm/models/Contact.java

```diff
@@ -4,13 +4,7 @@
 import com.onepagecrm.models.internal.CloseSalesCycle;
 import com.onepagecrm.models.internal.DeleteResult;
 import com.onepagecrm.models.internal.SalesCycleClosure;
-import com.onepagecrm.models.serializers.BaseSerializer;
-import com.onepagecrm.models.serializers.CloseSalesCycleSerializer;
-import com.onepagecrm.models.serializers.ContactPhotoSerializer;
-import com.onepagecrm.models.serializers.ContactSerializer;
-import com.onepagecrm.models.serializers.ContactSplitSerializer;
-import com.onepagecrm.models.serializers.DeleteResultSerializer;
-import com.onepagecrm.models.serializers.LoginSerializer;
+import com.onepagecrm.models.serializers.*;
```

Comment:
@lorobode 
This is not desirable. Better for size/memory to explicitly define each import individually.
I think you will need to change this in Android Studio / IntelliJ Idea settings.